src/app/modules/consultations/consultation-profile/consultation-questionnaire/consultation-questionnaire.component.ts

```diff
@@ -316,4 +318,32 @@ export class ConsultationQuestionnaireComponent implements OnInit, AfterViewInit
     });
   }
 
+  scrollToFirstInvalidControl() {
+    let errorElementFound;
+    const checkErrorElementExist = setInterval(() => {
```

Comment:
@Nishanthh Bro why is this setInterval there?

Reply:
@avikt couldn't find the error element instantly, handled with afterviewchecked